theme/static/js/hs-vue/subject-keywords-app.js

```diff
@@ -63,19 +67,20 @@ let subjKeywordsApp = new Vue({
             newVal.splice($.inArray(keywordName, this.resKeywords), 1);   // Remove the keyword
 
             this.error = "";
+            let vue = this;
 
             $.post("/hsapi/_internal/" + resIdShort + "/subject/add-metadata/",
                 {value: newVal.join(",")}, function (resp) {
                     if (resp.status === "success") {
-                        this.resKeywords = newVal;
+                        vue.resKeywords = newVal;
                         if (!newVal.length) {
                             // If no keywords, the metadata is no longer sufficient to make the resource public
                             manageAccessApp.onMetadataInsufficient();
                         }
                     }
                     else {
-                        if (this.resKeywords.length){
-                            this.error = this.message;
+                        if (vue.resKeywords.length){
+                            vue.error = resp.message;
```

Comment:
The shown error message is not very user friendly. Maybe we can simplify it with a one liner.

Reply:
Okay, I just modified so that it no longer uses the response error directly, but rather a custom error message within the js